packages/react-native-sdk/expo-config-plugin/src/withMainApplication.ts

```diff
@@ -0,0 +1,35 @@
+import { ConfigPlugin, withMainApplication } from '@expo/config-plugins';
+import {
+  addImports,
+  appendContentsInsideDeclarationBlock,
+} from '@expo/config-plugins/build/android/codeMod';
+
+const withStreamVideoReactNativeSDKMainApplication: ConfigPlugin = (config) => {
+  return withMainApplication(config, (config) => {
```

Comment:
```suggestion
const withStreamVideoReactNativeSDKMainApplication: ConfigPlugin = (configuration) => {
  return withMainApplication(configuration, (config) => {
```

to avoid the eslint warning

Reply:
done